src/index.html

```diff
@@ -6,13 +6,86 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="product-card__image"
+        />
+      </div>
+      <h1 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <h2 class="product-card__code">Product code: 195434</h2>
+
+      <div class="product-card__reviews">
+        <svg
```

Comment:
What is this svg doing here?
You should have used the same code as for the Stars task done before. The stars should be added using css and work exactly as the previous task (you can literally copy-paste most of it)